### DISCUSSION TITLE
$50 bounty: Only unblock for specified number of minutes after delayed.html countdown

### DISCUSSION THREAD

**Discussion by distbit0** [proginosko/LeechBlockNG#496] [Ideas]
Only unblock a blocking group for specified number of minutes after `delayed.html` countdown, instead of until the tab is closed

Hi I used to use the extension https://chromewebstore.google.com/detail/10-second-focus/pknpbhgcekdpigobealnbgjbeaonegad?hl=en before discovering leechblock. 

Leechblock is far superior in all ways to this extension except it is missing one feature which I found extremely useful. 

Leechblock doesn't seem to be able to resume blocking a "blocking group" after a specified number of minutes after the delay challenge was passed. What I mean is that I'd like the delayed page to allow me to specify how many minutes (ofc there can be a default value) I'd like to unblock this block group for, and then to re-block it automatically after that number of minutes has elapsed, even if I am still browsing a website included in the block group. I shouldn't have to refresh the tab in order for it to be re-blocked.  

As far as I can tell, leechblock doesn't currently support this functionality, although maybe I am just missing something due to being new. 

It would really help as otherwise I find myself scrolling social media feeds after unblocking e.g. twitter, even though I only intended to spend e.g. 10 minutes on twitter. 

many thx

**Comment by distbit0** [comment]
This idea was also proposed here: https://www.tapatalk.com/groups/leechblock/option-for-having-delaying-page-go-to-override-ins-t548.html

**Comment by proginosko** [comment]
An option along these lines will be included in the next release.

**Comment by distbit0** [comment]
Thanks so much! @proginosko . Lmk how you want to receive the $50.

**Reply by proginosko** [reply]
Wait and see if it works first!

**Reply by distbit0** [reply]
@proginosko will this also be released on the chrome version? many thx!

**Reply by proginosko** [reply]
> @proginosko will this also be released on the chrome version? many thx!

Yes!

**Reply by distbit0** [reply]
@proginosko 

pls provide payment details. thx

crypto address preferred, as I am outside the US, but I am open to suggestions ofc.  :)

**Reply by proginosko** [reply]
BTC: bc1qe4lv3l7340xaxj7xke2qgqev4ka6vdpa82e2zd

**Reply by distbit0** [reply]
Sent! Apologies for delay :)

**Reply by proginosko** [reply]
Appreciated!
